service/src/main/java/uk/nhs/adaptors/gp2gp/transformjsontoxmltool/TransformJsonToXml.java

```diff
@@ -68,7 +67,8 @@ public void run(String... args) {
             getFiles().forEach(file -> {
                 String xmlResult = mapJsonToXml(file.getJsonFileInput());
                 writeToFile(xmlResult, file.getJsonFileName());
-                xmlSchemaValidator.validateOutputToXmlSchema(file.getJsonFileName(), xmlResult);
+                ehrExtractMapper.validateXmlAgainstSchema(xmlResult);
```

Comment:
As PI Test shows below, this isn't required.  
Because the adaptor now throws the exception, and the transform tool just uses the application code, it will never reach this point if the validation has failed, so there will be no result XML to validate against